### PR TITLE
Updated numpy as np #4218 #5640

### PR DESCRIPTION
**Pull request by Amrithasuresh**
Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np from N

Updated numpy as np from N

Updated numpy as np from N

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np from N

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np from N

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Updated numpy as np

Removed np=numpy line=22

Updated numpy as np

Updated numpy as np

Updated numpy as np

removed 'change numpy to np

### CODE REVIEW

File: theano/tensor/elemwise.py [hunk 1]
```diff
@@ -504,7 +504,7 @@ def __setstate__(self, d):
         self.nfunc = None
         self.inplace_pattern = frozendict(self.inplace_pattern)
         if getattr(self, 'nfunc_spec', None):
-            self.nfunc = getattr(numpy, self.nfunc_spec[0])
+            self.nfunc = getattr(np, self.nfunc_spec[0])
         elif 0 < self.scalar_op.nin < 32:
             self.ufunc = np.frompyfunc(self.scalar_op.impl,
                                           self.scalar_op.nin,
```
Comment (Sentient07) [inline theano/tensor/elemwise.py]: Travis is failing with the following error. http://pastebin.com/au4MJiWV .  Fix all these `flake8` error messages. To ensure you have fixed them, run, `theano-nose theano/tests/test_flake8.py` locally. Also, it would be nice if commit messages are distinguishable.
Thanks

Comment (Amrithasuresh) [inline theano/tensor/elemwise.py]: @Sentient07 the conversations are [here](https://github.com/Theano/Theano/pull/5640). 

>Four or five files are conflicting often. It was hard for me to fix the files. I am learning now. So, the commit messages cannot be clear and it is messy. I guess it will be a headache for reviewing this PR. But, now I know how and where to do changes. In the weekend I will do a fresh fork and do fix the files one by one. That will be clear and easy.

Comment (Sentient07) [inline theano/tensor/elemwise.py]: IMO, the best fix for this situation would be, squash into one commit, move to a different branch(branching out from master), then cherry pick this single squashed commit to that branch. I would advise you to squash *only*  the commits that are adding imports. if you are doing something else in this PR, please keep it as a different commit.
